SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -56,7 +56,7 @@ NodeProxy : BusPlug {
 			if(freeGroup) {
 				oldGroup = group;
 				group = nil;
-				bundle.sched((fadeTime ? this.fadeTime) + (server.latency ? 0), { oldGroup.free });
+				server.sendBundle((fadeTime ? this.fadeTime) + (server.latency ? 0), [11, oldGroup.nodeID]);
```

Comment:
It's a little, little bit more efficient, but OTOH comments have the potential to get out of sync with code. Either way, I personally can't memorize all 62 commands :)